packages/x-virtualizer/src/models/dimensions.ts

```diff
@@ -66,6 +66,14 @@ export interface DimensionsState {
    * Size of the bottom container.
    */
   bottomContainerHeight: number;
+  /**
+   * Indicates that the container takes the height of its content.
+   */
+  autoHeight: boolean;
+  /**
+   * Minimal height to display the content area.
+   */
+  minimalContentHeight: string | number | undefined;
```

Comment:
These have been added in the model because the selectors need access to them, adding them was the easiest way to do that. The alternative would have been passing the params as selector arguments, but that complexifies the consumer API.